cmd/kubeadm/.import-restrictions

```diff
@@ -67,7 +67,6 @@
 				"k8s.io/kubernetes/pkg/kubelet/apis",
 				"k8s.io/kubernetes/pkg/kubelet/qos",
 				"k8s.io/kubernetes/pkg/kubelet/types",
-				"k8s.io/kubernetes/pkg/master/ports",
```

Comment:
The import boss is failing, because we still use this indirectly through various other packages. Let's just not remove it from the import-restrictions for now.

Reply:
Thanks for your help!